README.md

```diff
@@ -86,6 +86,9 @@ Contributions, issues, and feature requests are welcome!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- FUTURE FEATURES -->
```

Comment:
- [ ] As requested by the previous reviewer, Please kindly add at least one or two key features and future features sections to the README file to make it more professional. :information_source: